clang/CMakeLists.txt

```diff
@@ -650,6 +652,7 @@ if (CLANG_ENABLE_BOOTSTRAP)
     CLANG_VERSION_MAJOR
     CLANG_VERSION_MINOR
     CLANG_VERSION_PATCHLEVEL
+    CLANG_VERSION_SUFFIX
```

Comment:
There's currently nothing in clang/include/clang/Basic/Version.inc.in using this. Should we add it while we're here? Otherwise perhaps we should leave it out.

Reply:
This list is of CMake variables which automatically get passed through between stages in a multi-stage build, so it is useful here already, if someone specifies a custom CLANG_VERSION_SUFFIX. See docs in llvm/docs/AdvancedBuilds.rst

Reply:
D'oh, I made assumptions about the context without looking :/ All good then!